docs/api/net.md

```diff
@@ -63,6 +63,28 @@ Creates a [`ClientRequest`](./client-request.md) instance using the provided
 The `net.request` method would be used to issue both secure and insecure HTTP
 requests according to the specified protocol scheme in the `options` object.
 
+### `net.fetch(input[, init])`
+
+* `input` RequestInfo
+* `init` RequestInit (optional)
```

Comment:
question: are there any good links to the structure of `RequestInfo` or `RequestInit` aside from the Fetch standard itself? https://fetch.spec.whatwg.org/#requestinfo

The MDN doc on fetch doesn't seem to mention those two structures explicitly (https://developer.mozilla.org/en-US/docs/Web/API/fetch), although I imagine the parameters detailed under `resource` and `options` are the same as `RequestInfo` and `RequestInit`, respectively.

Reply:
`RequestInfo` is defined as `string | Request`, so we could specify that instead, as it's fairly straightforward. The [`RequestInit` structure](https://fetch.spec.whatwg.org/#requestinit) is much more complex, and as you noted MDN doesn't document it as such. Perhaps we could link https://developer.mozilla.org/en-US/docs/Web/API/fetch#options?

Reply:
Added some links.